plugins/inputs/nginx_plus_api/nginx_plus_api_metrics_test.go

```diff
@@ -536,7 +536,7 @@ const streamUpstreamsPayload = `
 			"state": "up",
 			"active": 0,
 			"max_conns": 30,
-			"connecions": 1231,
+			"connections": 1231,
```

Comment:
Same here, are you sure the API responds it like this?

Reply:
> Same here, are you sure the API responds it like this?

No. Reverted the change in order to avoid a regression.